main/src/OnACoffeeLoop.java

```diff
@@ -0,0 +1,214 @@
+import java.util.Scanner;
+
+public class OnACoffeeLoop {
+
+    //declaring our class variables
+    static int water = 400;
+    static int milk = 540;
+    static int coffeeBeans = 120;
+    static int disCups = 9;
+    static int money = 550;
+    static String buy = "buy";
+    static String fill = "fill";
+    static String take = "take";
+    static String espresso = "1";
+    static String latte = "2";
+    static String cappuccino = "3";
+    static String exit = "exit";
+    static String remaining = "remaining";
+    static String back = "back";
+
+
+    public static void action ()  // method action asks to user what he can do (buy, fill, take, see remaining,or exit)
```

Comment:
@Edvard777 why you don't use parameters , and making everything class level

Reply:
done